src/ws/frame.rs

```diff
@@ -372,6 +374,10 @@ impl fmt::Display for Frame {
     }
 }
 
+/// A `Binary` representing a `WebSocket` message with framing.
```

Comment:
I think it would be better to say just:

>`WebSocket` framed message

We don't really need to overload users :)

Reply:
updated